### PR TITLE
Improve checkKeys error message

### PR DESCRIPTION
**Pull request by DavidBruant**

### CODE REVIEW

**Review by katelynsills — APPROVED** [review]
Looks good to me. thanks!

**Review by katelynsills — APPROVED** [review]
Oh actually let's make sure the tests pass

**Review by katelynsills — CHANGES_REQUESTED** [review]
Sorry, this should be "request changes" because the tests aren't passing

**Comment by katelynsills** [comment]
I think this just has linting errors. If we fix those, we can merge

File: packages/zoe/src/cleanProposal.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const checkKeys = (expectedKeys, record) => {
     assert.typeof(key, 'string');
     assert(
       expectedKeys.includes(key),
-      details`key ${key} was not an expected key`,
+      details`key ${key} was not one of the expected keys (${expectedKeys.join(', ')})`,
```
Comment (Chris-Hibbert) [inline packages/zoe/src/cleanProposal.js]:
```suggestion
      details`key ${key} was not one of the expected keys (${expectedKeys.join(
        ', ',
      )})`,
```

**Review by Chris-Hibbert — APPROVED** [review]
eslint wanted a change in the format, but otherwise, this looks like an improvement.

**Comment by katelynsills** [comment]
@DavidBruant I'd love to merge this. Do you think we could fit it in this week?

**Comment by DavidBruant** [comment]
yes; i'll fix the conflict

**Comment by DavidBruant** [comment]
Things moved after the sparse keyword PR
I'm closing this PR in favor of #873

**Pull request closed without merging**